server/jobs/moss.py

```diff
@@ -10,7 +10,7 @@
 from server import jobs
 
 @jobs.background_job
-def submit_to_moss(moss_id=None, file_regex="*", assignment_id=None, language=None):
+def submit_to_moss(moss_id=None, file_regex=".*", assignment_id=None, language=None):
```

Comment:
Kind of, unfortunately it needs to be changed in three places. forms.py, templates/staff/jobs/moss.html, and here I guess.